### PR TITLE
So much for that small PR i was working on.

### LINKED ISSUE #4729: Mapping: guild reception is missing ID lock.
Simple as. Should have A interior windoor with a lock. Most importantly it will protect adept blattin from the uninformed newbies.

### LINKED ISSUE #4731: Mapping: robotics go boom (again)
This was a issue on are old map. We FIXED it then went and did it again on the new map. There is a welding fuel tank lined up to get shot by a vender and EN being ideal target for it. An easy fix is relocating the welding fuel tank to the back storage room where the ADMS lives.

dreamseeker_kHcGGZoG9Q

My suggested location: 
dreamseeker_JbKlAStnMz

### PR DESCRIPTION
**Pull request by cdb-is-not-good**
maps in hibikleaner Morikroomba, they've been captured by those dastardly commies.

slightly tweaks the difficulty of the excelsior mini-dungeon.

adds in random  mob-spawner that picks between excelsior vatgrowns or a dead ass mindslave at a rate of about 50/50 alive:dead.

that one welding tank in robotics that kept blowing up has been moved to the R&D lab, im sure yall can share. Fixes #4731

adds a second windoor into the guild front desk, this one has the same access reqs as the front door(so folks cant steal blattedin). Fixes #4729

Fixes a small bug where the WO Officer/Interrogation room buttons had too much range and would trigger electrochromatic windows in the next room over

At the request of Dirk, janitor shutters given the same access as the rear entry door

Fixes a small issue with missing conveyor IDs on the ore-processing train in the upper level of lonestar mining prep. Removes an erronious wall table in R&D

Fixes the cryo-bed in isolation not having a console(and thus not working.)

Adds a few more random trapdoor spawns that were missing from the surface.

Difficulty adjusts lower-maints. This area is meant to be /dangerous/ and has been adjusted. To compensate, a few more rare loot spawns have been added to the mini-dungeons..

Maints further dangerified with the return of traps. Chances are low for any single door to have a trap, but one must always be careful.

Biomass fixed for real. Now causes chemical burns on targets that get too close without bio protection- corrosive fumes, i guess? standard spawns in maints only affect you when walking onto/over them, whereas spawns from the biomass cleanups in the bioreactor are massively lethal to anything anywhere near them.

Pararein buffed. Not quite as deadly as it used to be, but still rather bad to be bit.
turns out str 2 toxin does nothing, still. raises strength, slows down metabolism so it sticks in the system like a more traditional toxin.

Maintenance now has a number of APCs mapped in, wired but turned off. If you feel up to the task, take to the tunnels like the adepts of old and get that power cranked - or tear the things up when drifters start sucking power from your net! Be careful, however- The wires connecting the APCs are a bit old and exposed wire may have begun to fray exposing mainline voltages to open air.

Also adds a few lights in maintenance in areas that had not had them- presumably due to not having power till now. tsk tsk, bad level design no?

Fixes sts-40 missing sprite

Buffs the Ostwind a bit. given a bit more firerate - should now be worth its diamond cost.

Nerfs the ostwind a bit - 3 max mods instead of 5.(Im honestly kinda sitting here wondering if any guns should have 5 mods max, it's a bit ridiculous)

Spec has lost the ability to choose the spec-op as a starting weapon, it's just too good and based. Ranger is keeping it *for now*, but primarily because they have a very middling lineup without a lotta options

More tweaks to toxins, this time across the board. Sparing the boring stuff, toxicity now goes *up* with dose rather than being a static number set by the chem(OD already did this and still does but it FAR more harsh.).

For those that care about the nitty gritty- before this change, ce_toxin=strength. CE_TOXIN<10~ did...basically nothing, which meant that toxins did nothing unless overdosing(OD causes strength get multiplied by dose/4. 32 pararein at strength 6 becomes 42 which is giant). Now, dose directly causes strength to go up. Rather than being multiplicative as in OD it is additive. toxicity=strength+dose/2. 20 pararein at strength 4 becomes 14. For a bit of useful info, at 12 toxicity a liver will begin taking toxin wounds and in about 20 minutes will start taking brute damage(on my test server anyway.) once the liver is dead, poison starts attacking the rest of the organs resulting probably in MSOF.

Im sure i know what you're thinking, "CDB that sounds REALLY deadly!" not so my handsome reader, a single dose of dylovene will not only purge your blood of the poison but will also clear the damage to the liver(this will likely be subject to a balance pass later if im happy with the toxin changes.)

Kaisers have been(hopefully) returned to the status of leviathan. Kaisers are now notably faster, around the same speed as their smaller kin(notably, their preference to stay at range and their speed mean that they are relatively easy to disengage from in light of their speed.) This however, is not what makes them dangerous. 

Their spit, like many toxin based damage sources wasn't working. In keeping with its history as an irradiating goo it now inflicts burns, as well as leaving behind a pile of HIGHLY radioactive goo that will affect anyone foolish enough to stay anwhere near said gloop

Radiation has also been updated to actually function. Low levels of radiation incur occasional  burns/genetic damage. From the testing I did it SEEMS relatively forgiving until you hit like, 75% + radiation

Marshal armor kits tweaked at the request of 2/3 the only active marshal players.

Armored overcoat is back in the loadout vendor- comes with a coat and standard non-full helmet.

Webbing vest back in the loadout vendor, comes with a full-helm.

Marshal standard armor given a full helm to better match the armor, standard plate starts as a full-plate rather than just a vest

webbing vest buffed slightly, matches the standard plate but lacks the ability to offer full coverage. Also nerfed slightly, comes with 0.1 slowdown(about the same as a plate carrier.)

Some slightly new sprites from Eris for bulletproof armor/helmet, riot armor.

marshal greatcoat given alternative 'field jacket' icon, courtesy of Eris.

ports those syndicate suits from Eris to match the berets, tosses both in the loadout under beret/turtleneck selectors. Of course I mean the Xanoranth syndicate, who else would i mean? A solid pick for all your voidwolf drip needs

Turns out all that 'tweaking' i did to radiation was being done to the wrong path(it was for brains?). Thanks to Trilby, actually fixes it. Radiation, as previously stated - will cause occasional burns and genetic damage that gets higher with rad levels

Slightly raises the kaisers armor(slows it down a bit as well.)

Ameridian runtling speed lowered from 1-3

Ameridian beheamoth speed lowered from 3-5

Bloodred and CE hardsuits had some- "old" sprites, helmets and for the Blood Red gauntlets(which didnt....match and exposed the wrists.). They've been updated to modern matching sprites(Credit to Eris for said sprites)

reduces the amount of effective range pistol, calibers have by 1-2 each. 

I forget who asked me to do this, but butchers the solfed trooper sprites for their helmets, then adds said helmets as alternates for the SCAF helm - for all your surplus soyfed oorah larping needs. - OG sprites apparently by rebel?

Makes the tomahawk small(like the crovel, it folds up.)
at the requests of a few players, nerfs the sunrise - it may no longer be modified in light of its already high baseline performance

The greyson combat suit is now a subtype of the Void style suit and comes with its helmet as apart of the kit

Fixes a bug with the CO coat missing some sprites. Removes the non-IFF'd CO coat from selection.

maybe nerfs pararein abit

## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

	
<hr>
</details>

## Changelog
:cl:

/:cl:

### CODE REVIEW

File: code/modules/reagents/reagents/toxins.dm [hunk 1]
```diff
@@ -515,6 +515,7 @@
 	M.stats.addTempStat(STAT_VIG, -STAT_LEVEL_BASIC * effect_multiplier, STIM_TIME, "pararein")
 	M.stats.addTempStat(STAT_TGH, -STAT_LEVEL_BASIC * effect_multiplier, STIM_TIME, "pararein")
 	M.stats.addTempStat(STAT_COG, STAT_LEVEL_ADEPT * effect_multiplier, STIM_TIME, "pararein")
+	M.add_chemical_effect(CE_TOXIN, strength)
```
Comment (Trilbyspaceclone) [inline code/modules/reagents/reagents/toxins.dm:518]: wew getting hit with 6 strength twice

Comment (cdb-is-not-good) [reply code/modules/reagents/reagents/toxins.dm:518]: does it do both this act_effect AND the parent?

Comment (Trilbyspaceclone) [reply code/modules/reagents/reagents/toxins.dm:518]: yes

Comment (cdb-is-not-good) [reply code/modules/reagents/reagents/toxins.dm:518]: lol oops, i'll- go ahead and take away this one

**Pull request merged**